deployment-guide.md

```diff
@@ -19,98 +19,47 @@ You can deploy your contracts and run an end-to-end test or demo as follows:
 
     Once anvil is started, keep it running in the terminal, and switch to a new terminal.
 
-2. Deploy an `IBonsaiRelay` contract by running:
+2. Set your private key:
 
     ```bash
-    RISC0_DEV_MODE=true forge script script/Deploy.s.sol --rpc-url http://localhost:8545 --broadcast
+    export ETH_WALLET_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
     ```
 
-3. Check the logs for the address of the deployed `BonsaiTestRelay` contract and your application contract.
-   Save them to a couple of environment variables to reference later.
+3. Deploy your contract by running:
+    > This requires that `ETH_WALLET_PRIVATE_KEY` be set.
```

Comment:
Is this the only way to provide a private key? It seems ok for the template to only provide one way, but I imagine some developers will need to revise this to use a wallet that it controlled in a way where they don't have access to the private key. It seems alright to me that those developers will need to work with `forge` and the deploy script to make this happen.